modin/pandas/test/test_series.py

```diff
@@ -1905,6 +1905,30 @@ def test_index(data):
     df_equals(modin_series.index, pandas_series.index)
 
 
+@pytest.mark.parametrize("name", ["Dates", None])
+@pytest.mark.parametrize(
+    "dt_index", [True, False], ids=["dt_index_true", "dt_index_false"]
+)
+@pytest.mark.parametrize(
+    "data",
+    [*test_data_values, "empty"],
+    ids=[*test_data_keys, "empty"],
+)
+def test_index_dtype(name, dt_index, data):
```

Comment:
Why did you copy `test___repr` to test the case? You only needed to add the line `df_equals(modin_series.index, pandas_series.index)` after that line because the issue persisted for BaseOnPython only. https://github.com/modin-project/modin/blob/c6393b54639b75ad38f028ba06c6eb1e43bfb790/modin/pandas/test/test_series.py#L508

Also, create an issue for this PR, please.

Reply:
@YarShev We have issue about default dtypes: https://github.com/modin-project/modin/issues/3142

Reply:
I know. We follow the following practice: for each PR should be created an issue to be resolved.

Reply:
@YarShev Thanks!